astropy/tests/pytest_doctestplus.py

```diff
@@ -11,10 +11,7 @@
 
 from .output_checker import AstropyOutputChecker, FIX
 
-try:
```

Comment:
Potential conflict with #6606, probably better to leave it or fix it upstream. 

@astrofrog suggested the same to me in #6641.